build.gradle

```diff
@@ -22,12 +22,16 @@ buildscript {
         // 2.0.0-alpha1-SNAPSHOT -> 2.0.0.0-alpha1-SNAPSHOT
         version_tokens = opensearch_version.tokenize('-')
         opensearch_build = version_tokens[0] + '.0'
+        job_scheduler_no_snapshot = opensearch_build
         if (buildVersionQualifier) {
             opensearch_build += "-${buildVersionQualifier}"
+            job_scheduler_no_snapshot += "-${buildVersionQualifier}"
         }
         if (isSnapshot) {
             opensearch_build += "-SNAPSHOT"
         }
+        opensearch_no_snapshot = opensearch_version.substring(0, opensearch_version.indexOf("-", opensearch_version.indexOf("-") + 1))
```

Comment:
If this is just to remove the `-SNAPSHOT`, can't we just persist the `opensearch_build` var as a separate var before the optional `-SNAPSHOT` is added on lines 30-32?

Reply:
In fact, what's the difference between `job_scheduler_no_snapshot` and `opensearch_no_snapshot`?

Reply:
Just replace `-SNAPSHOT` with ` ` should be enough.

Reply:
job_scheduler_no_snapshot = 2.0.0.0-alpha1
opensearch_no_snapshot = 2.0.0-alpha1
The optional -SNAPSHOT is added after adding the `.0`
@peterzhuamazon good call, I'll change to `opensearch_version.replace("-SNAPSHOT, "")`

Reply:
@ohltyler since one has the .0 and one doesn't, I changed it to `.replace` from the `opensearch_version`. Might have misunderstood what you meant

Reply:
Yeah, missed that. Your change looks good to me!